src/Iyzipay/Request/CreateRefundRequest.php

```diff
@@ -12,6 +12,8 @@ class CreateRefundRequest extends Request
     private $price;
     private $ip;
     private $currency;
+    private $reason;
```

Comment:
We can add reason enum values like Locale,Currency, etc.

Reply:
You can't do it in PHP. Language has type inference feature.